EU4ToVic2Tests/MapperTests/TechSchoolTests.cpp

```diff
@@ -0,0 +1,115 @@
+#include "gtest/gtest.h"
+#include "../EU4toV2/Source/mappers/TechSchools/TechSchool.h"
+
+
+TEST(Mappers_TechSchoolTests, MatchingInvestmentCausesScoreOfZero)
+{
+	std::istringstream input("");
+	mappers::TechSchool newTechSchool(input);
+	ASSERT_EQ(0, newTechSchool.calculateComparisonScore(0.2,0.2,0.2,0.2,0.2));
+}
+
+TEST(Mappers_TechSchoolTests, IncreasingArmyInvestmentIncreasesScore)
+{
+	std::istringstream input("");
+	mappers::TechSchool newTechSchool(input);
+	ASSERT_LT(0, newTechSchool.calculateComparisonScore(0.3,0.2,0.2,0.2,0.2));
+}
+
+TEST(Mappers_TechSchoolTests, DecreasingArmyInvestmentIncreasesScore)
+{
+	std::istringstream input("");
+	mappers::TechSchool newTechSchool(input);
+	ASSERT_LT(0, newTechSchool.calculateComparisonScore(0.1,0.2,0.2,0.2,0.2));
+}
+
+TEST(Mappers_TechSchoolTests, ArmyBonusImported)
+{
+	std::istringstream input("{\n\t\tarmy_tech_research_bonus=0.1}");
+	mappers::TechSchool newTechSchool(input);
+	ASSERT_GT(0.0001, abs(newTechSchool.calculateComparisonScore(0.3,0.2,0.2,0.2,0.2)));
```

Comment:
I don't recall exactly how this works, but gtest has ASSERT_NEAR if you're trying to handle floating point comparison: https://github.com/google/googletest/blob/master/googletest/docs/advanced.md